docs/reference/argocd.md

```diff
@@ -972,6 +974,7 @@ Name | Default | Description
 Host | example-argocd | The hostname to use for Ingress/Route resources.
 [Ingress](#server-ingress-options) | [Object] | Ingress configuration for the Argo CD Server component.
 Insecure | false | Toggles the insecure flag for Argo CD Server.
+Replicas | 1 | The number of replicas  for Argo CD Server. If Server Autoscale is enabled, the number of replicas defined here will be overriden by the number of replicas decided by the autoscaler. 
```

Comment:
Default to nil. If this value set, it must be >= 0.  This value is passed to replicas in Deployment spec.  If HA is enabled, this field is ignored.